Doc/library/sqlite3.rst

```diff
@@ -367,22 +369,27 @@ Module functions
    to disable the feature again.
 
    Register an :func:`unraisable hook handler <sys.unraisablehook>` for an
-   improved debug experience::
+   improved debug experience:
+
+   .. testsetup:: sqlite3.trace
```

Comment:
You should be able to use `.. testsetup:: *` to apply this setup to all groups and avoid duplication.

Reply:
This is deliberate, as we don't want to run the other tests with a (malformed) trace callback. Alternatively, we could create hidden setup/cleanup code for this particular test, but I find using a separate namespace is a clean approach.

Reply:
FYI: the trace callback is triggered for _every_ executed SQL statement.

Reply:
I reviewed from mobile and it's a bit cramped here, so I might have missed something.  I meant that you could use a global setup that just imports `sqlite3`, so that you don't need to repeat it elsewhere 

If in some examples you want an explicit and visible `import sqlite3`, you can still add it and it should work fine even if it was already imported.